guppylang/std/quantum.py

```diff
@@ -34,7 +33,7 @@ def measure(self: "qubit" @ owned) -> bool:
     @guppy
     @no_type_check
     def measure_return(self: "qubit") -> bool:
-        return measure_return(self)
+        return project_z(self)
```

Comment:
Just a comment, no change needed. From this wrapper, it looks like `project_z` is not exposed directly to the user, so I think it's a nice rename. I wouldn't want users to have to use `project_z` call for measurements.